### ISSUE TITLE
Case sensitivity with SQLite

### ISSUE THREAD

**Issue by wbyoung** [wbyoung/azul#14]
Case sensitive queries on SQLite are currently treated like case insensitive queries. So the following:

```js
Article.objects.where({ 'title[contains]': 'news' })
```

Is really the same as a query that's executed with `icontains`.

Before we can address this, mapbox/node-sqlite3#140 will need to be fixed.

**Comment by wbyoung** [comment]
Another issue that may have some relevant information is mapbox/node-sqlite3#93.
